libs/blocks/modal/modal.js

```diff
@@ -195,6 +207,7 @@ export function getHashParams(hashStr) {
 export function delayedModal(el) {
   const { hash, delay } = getHashParams(el?.dataset.modalHash);
   if (!delay || !hash) return false;
+  isDelayedModal = true;
```

Comment:
should you set `isDelayedModal` to false after closing or better after opening it?